crates/wit-bindgen-wasmcloud-provider/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "wit-bindgen-wasmcloud-provider"
+version = "0.1.0"
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+publish = false
+description = """
+Bindgen macro for binary-based wasmCloud capability providers written in Rust
+"""
+
+[lib]
+proc-macro = true
+
+[dependencies]
+anyhow = { workspace = true }
+heck = { workspace = true }
+proc-macro2 = { workspace = true }
+quote = { workspace = true }
+serde = { workspace = true }
+syn = { version = "1.0.92", features = [ "parsing", "full", "visit-mut", "extra-traits" ] }
```

Comment:
Is there a reason to use a different version from `2`, which is already in the workspace? Could you add a comment here describing it, if there is one? (and file an issue to use a consistent version?)

Reply:
Yes, there is a reason, otherwise the version would not be `1.0.92` -- will add comment here so curious people can tell at a glance.

- [ ] Add comment around `syn` v1.0.92` version